yhondanilo/chess.rb

```diff
@@ -0,0 +1,41 @@
+class Chess
+
+	attr_accessor :fila, :columna
+
+	def initialize(fila, columna)
+		@fila, @columna=fila, columna
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Surrounding space missing for operator '='.<br>Tab detected.